python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_decorator.py

```diff
@@ -270,3 +273,40 @@ def my_dbt_assets():
         AssetKey(["raw_orders"]): "default",
         AssetKey(["raw_payments"]): "default",
     }
+
+
+def test_dbt_with_downstream_asset_errors():
+    @dbt_assets(manifest=test_dagster_metadata_manifest)
+    def my_dbt_assets():
+        ...
+
+    with pytest.raises(
+        DagsterInvalidDefinitionError,
+        match=(
+            "Cannot pass a multi_asset AssetsDefinition as an argument to deps."
+            " Instead, specify dependencies on the assets created by the multi_asset via AssetKeys"
+            " or strings."
+            " For the multi_asset my_dbt_assets, the available keys are: "
```

Comment:
really struggling to get the regex match to work on the possible asset keys, so leaving this blank for now. might take another stab at the match tomorrow